### ISSUE TITLE
视频下载后无法播放

### ISSUE THREAD

**Issue by angelh1209** [xifangczy/cat-catch#72]
**bug描述**下载后的视频 码率信息不正常 打开显示需要encv解码器  
![图片](https://user-images.githubusercontent.com/101373674/190277902-fc6c1f0c-3bab-4a07-9fe8-a4618899668e.png)

**扩展**
 - 版本号2.2.4

**安装方式**
- 官方扩展店
- 源码安装

**涉及网址**https://www.mewatch.sg/watch/Taste-of-Life-E17-97028

**Comment by angelh1209** [comment]
怎么感觉视频还是被加密的

**Comment by xifangczy** [comment]
是的 视频有DRM。你可以尝试 [N_m3u8DL-CLI](https://github.com/nilaoda/N_m3u8DL-CLI) 或 [N_m3u8DL-RE](https://github.com/nilaoda/N_m3u8DL-RE)  工具试试。
我的新加坡代理速度很慢 没仔细研究，一般dash支持多种DRM方案，通常解密成功率都不高。

**Comment by angelh1209** [comment]
我去试试。速度慢可以在浏览器内用插件
![图片](https://user-images.githubusercontent.com/101373674/190386346-26bf6484-8fe6-484e-abe6-9c46b0ad92e2.png)
 切换新加坡IP 1.32.191.255
